src/components/markdown-renderer/replace-components/vega-lite/vega-chart.tsx

```diff
@@ -1,7 +1,7 @@
 import React, { Fragment, useCallback, useEffect, useRef, useState } from 'react'
 import { Alert } from 'react-bootstrap'
 import { useTranslation } from 'react-i18next'
-import embed, { VisualizationSpec } from 'vega-embed'
+import { VisualizationSpec } from 'vega-embed'
```

Comment:
Doesn't this type need to lazy loaded, too?

Reply:
No, because it's a type. It's only relevant during the typecheck. An import doesn't import the hole file.